node/lib/DeviceAccessor.ts

```diff
@@ -1,18 +1,89 @@
 
 import { DeviceInterface } from "./DeviceInterface";
-import { IDevice } from "./ITranslator";
+import { ITranslator } from "./ITranslator";
 
 import { EventEmitter } from "events";
 
 /**
  * Provides reflection-style access to device properties and methods via device interfaces.
  */
 export class DeviceAccessor {
+    /**
+     * Loads a DeviceInterface from a module. This is just a convenience wrapper
+     * around require(). Throws if the module could not be loaded.
+     *
+     * @param {string} interfaceModuleName  Package-qualified name (or relative path) of
+     *     the interface module. The package-qualified name can be obtained from
+     *     PackageTranslatorInfo.interfaces.
+     *
+     * @returns {Promise<DeviceInterface>} The loaded device interface.
+     */
+    public static getInterfaceAsync(interfaceModuleName: string): Promise<DeviceInterface>;
```

Comment:
>getInterfaceAsync(interfaceModuleName: string): Promise<DeviceInterface>; [](start = 18, length = 73)

Just to validate my understanding, do these methods need to be changed to return list of DeviceInterface objects when 'composability at manifest level' is allowed ?